raiden/tests/integration/test_raidenservice.py

```diff
@@ -113,10 +97,8 @@ def test_broadcast_messages_must_be_sent_before_protocol_messages_on_restarts(
     # Asserts the balance proofs are broadcasted before protocol messages
     def start_transport(*args, **kwargs):
         # Before restart the transport's broadcast queue should be initialized
-        # There should be 3 messages in the queue:
-        # - A `MonitorRequest` to the MS
-        # - A `PFSCapacityUpdate`
-        # - A `PFSFeeUpdate`
+        # There should be A `MonitorRequest` message to the MS in the queue:
+        # -
```

Comment:
Why aren't the other updates sent any more?

Reply:
These updates were sent as part of the initialization service, and this PR is introducing the idea of setting up channel fees via the API and no longer on startup.